packages/core/getAggregatedAvailability.ts

```diff
@@ -22,25 +22,33 @@ export const getAggregatedAvailability = (
   return mergeOverlappingDateRanges(availability);
 };
 
+function isSameDay(date1: Date, date2: Date) {
+  return (
+    date1.getUTCFullYear() === date2.getUTCFullYear() &&
+    date1.getUTCMonth() === date2.getUTCMonth() &&
+    date1.getUTCDate() === date2.getUTCDate()
+  );
+}
+
 function mergeOverlappingDateRanges(dateRanges: DateRange[]) {
-  const sortedDateRanges = dateRanges.sort((a, b) => a.start.diff(b.start)); //is it already sorted before?
+  dateRanges.sort((a, b) => a.start.valueOf() - b.start.valueOf());
```

Comment:
Biggest win by far, before the performance showed >100ms for my small-ish event type, after it showed 0.1ms.